src/components/Proudects.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useEffect, useState,useContext } from 'react'
+import {ProductContxt} from '../App'
+import axios from 'axios'
+import '../style/Products.css'
+function Proudects() {
+    const [products, SetData] = useState();
+    const {clicked, setClicked} = useContext(ProductContxt);
+    useEffect(() => {
+        axios.get('https://fakestoreapi.com/products')
+            .then((res) => {
+                SetData(res.data)
+            })
+            .catch((err) => SetData(err))
+    }, [])
+    const jsx = products?.map((e) =>
```

Comment:
Why is it named jsx? Renaming it to "products" makes more sense

Reply:
In addition, this whole block:
```
<div key={e.id} className='pro' >
            <img src={e.image} className='pro_image'/>
            <div className='pro_contain'>
                <span className='pro_rating'>{e.rating.rate}</span>
                <span className='pro_count'>{e.rating.count} Rating</span>
                <h3 className='pro_title'>{e.title}</h3>
                <span className='pro_price'>${e.price}</span>
                <button className='pro_btn' onClick={()=>{setClicked(prev=>prev+1)}}>Add to cart</button>
            </div>
```

can be moved to a separate component in a seperate file called Product.js

Then map it as so
`const products = products?.map((product) =><Product name={product.name} rating={product.rating} etc... />)`

Reply:
the product would be empty ???

Reply:
done